docs/content/en/docs/pipeline-stages/builders/docker.md

```diff
@@ -67,15 +67,37 @@ These credentials are configured in the `cluster` section with the following opt
 
 {{< schema root="ClusterDetails" >}}
 
-To set up the credentials for Kaniko refer to the [kaniko docs](https://github.com/GoogleContainerTools/kaniko#kubernetes-secret) (**Note**: Rename the downloaded JSON key to *kaniko-secret* without appending *.json*).
-Alternatively, the path to a credentials file can be set with the `pullSecret` option:
+### Configure Kaniko Credentials 
+
+To set up the credentials for Kaniko refer to the [kaniko docs](https://github.com/GoogleContainerTools/kaniko#kubernetes-secret)
+
+(**Note**: Rename the downloaded JSON key to *kaniko-secret* without appending *.json*).
+
+Alternatively, the path to credentials file can be set with the `pullSecretPath` option:
 ```yaml
 build:
   cluster:
     pullSecretName: pull-secret-in-kubernetes
-    # OR
-    pullSecret: path-to-service-account-key-file
+    pullSecretPath: path-to-service-account-key-file-within-secret
+  
+```
+
+Skaffold can also set up credentials if a secret does not exist in your cluster. This is usually the case when you are performing a 
+build in your personal cluster. 
```

Comment:
To me, `personal cluster` suggests Minikube or something similar.

Reply:
it could be your personal (local or remote) GKE cluster  In case developers are using. 
Is there an umbrella term for that?

Reply:
I think it would be helpful to explicitly highlight that this is for GKE/GCR?

Reply:
Kaniko can also on EKS, Azure hence i mentioned personal cluster. 
What i want to focus on is, usually if you are in a shared cluster env, secrets will be installed by admins and developers don't have access to create them

However, when you are using a personal cluster to deploy or in personal namespace on a shared cluster, you would need to create you own secret.

Reply:
maybe reword to :
```
Skaffold can also set up credentials if a secret does not exist in your cluster. This is usually the case when you are not using a shared cluster where the cluster admin would have already configured it for you. 
```